content/en-us/reference/engine/classes/ScrollingFrame.yaml

```diff
@@ -16,7 +16,7 @@ properties:
     summary: |
       The size in offsets of the area that is scrollable.
     description: |
-      The size in offsets of the area that is scrollable.
+      The size in offsets of the area that is scrollable. This property is set to the maximum of the `Class.ScrollingFrame.CanvasSize` property and the size of the children if `Class.ScrollingFrame.AutomaticCanvasSize` is set to something other than `None|Enum.AutomaticSize`.
```

Comment:
```suggestion
      The size in offsets of the area that is scrollable. By default, this property is
      set to the maximum of the `Class.ScrollingFrame.CanvasSize|CanvasSize`
      property and the size of the children if
      `Class.ScrollingFrame.AutomaticCanvasSize|AutomaticCanvasSize` is set
      to something other than `Enum.AutomaticSize.None`.
```

Reply:
I think  "By default" is not necessary here since this is the only behavior (the property is read only)

Reply:
Thanks! I removed "By default" and I'll merge now.